src/svc_auth_gss.c

```diff
@@ -406,13 +397,16 @@ _svcauth_gss(struct svc_req *req, bool *no_dispatch)
 	OM_uint32 min_stat;
 	bool gd_locked = false;
```

Comment:
You're not using gd_locked anymore.  You can remove it.

Reply:
Good suggestion. Removed it